mkDocs.sh

```diff
@@ -124,7 +124,7 @@ if [[ ! -f ${BASEDIR}/target/generated-docs/typescript-doc/classes/_api_.writeap
     echo 'File "_api_.writeapi.md" is not generated, aborting.'
     exit -1
 fi
-if [[ ! -f ${BASEDIR}/target/generated-docs/reference.md ]] ; then
-    echo 'File "reference.md" is not generated, aborting.'
+if [[ ! -f ${BASEDIR}/target/generated-docs/tagger_reference.md ]] ; then
+    echo 'File "tagger_reference.md" is not generated, aborting.'
```

Comment:
IMO past tense sounds better, i.e. `File [...] was not generated`

Reply:
Done